rosbag2_cpp/include/rosbag2_cpp/writers/sequential_writer.hpp

```diff
@@ -115,7 +116,7 @@ class ROSBAG2_CPP_PUBLIC SequentialWriter
   std::unique_ptr<Converter> converter_;
 
   bool use_cache_;
-  std::shared_ptr<rosbag2_cpp::cache::MessageCache> message_cache_;
+  std::shared_ptr<rosbag2_cpp::cache_interfaces::BaseMessageCacheInterface> message_cache_;
```

Comment:
Perhaps it could be shortened as:
```suggestion
  std::shared_ptr<MessageCacheInterface> message_cache_;
```